### ISSUE TITLE
github module stucks dashboard

### ISSUE THREAD

**Issue by berhram** [wtfutil/wtf#1460]
## What's broken?
When using the GitHub module and pressing the scroll key multiple times, the app is stuck and crushed.
## Additional info:
OS: Fedora Linux 37 (Workstation Edition) x86_64
Installed from: homebrew/core

**Comment by Seanstoppable** [comment]
Yeah, this is unfortunately a known issue if you are switching windows several times in quick succession. We haven't quite figured out root cause yet, but will try to dive in again.
